packages/mars-theme/src/html/components/theme.js

```diff
@@ -30,6 +31,7 @@ const Theme = ({ state }) => (
     <Body>
       {state.source.data(state.router.path).isArchive && <List />}
       {state.source.data(state.router.path).isPostType && <Post />}
+      {state.source.data(state.router.path).is404 && <Page404 />}
```

Comment:
I tried this yesterday and `is404` was undefined when I entered a wrong url. When exactly is `is404` populated?

Reply:
What url did you try? The property `is404` is set [here](https://github.com/frontity/frontity/blob/dev/packages/wp-source/src/actions.ts#L35), when a handler throws an exception.

Reply:
I tried something invented like `/invented-path/` or something like that. Maybe the error was somewhere else and I was mistaken?

Reply:
I don't know, but it seems to be working fine now.